illumos-utils/src/addrobj.rs

```diff
@@ -5,7 +5,17 @@
 //! API for operating on addrobj objects.
 
 /// The name provided to all link-local IPv6 addresses.
-pub const IPV6_LINK_LOCAL_NAME: &str = "ll";
+pub const IPV6_LINK_LOCAL_ADDROBJ_NAME: &str = "ll";
+
+/// The name provided to all static IPv6 underlay addresses.
+pub const IPV6_STATIC_ADDROBJ_NAME: &str = "omicron6";
+
+/// The name provided to all static IPv4 addresses, usually for public OPTE
+/// interfaces.
+pub const IPV4_STATIC_ADDROBJ_NAME: &str = "omicron6";
```

Comment:
I believe this should be `omicron4` instead

```suggestion
pub const IPV4_STATIC_ADDROBJ_NAME: &str = "omicron4";
```

Reply:
Yeah, good catch, thanks!

Reply:
Done in 6c93f59a4